docs/virtualenv_guide.md

```diff
@@ -23,6 +23,18 @@ sudo apt-get install python3.6
 brew install python
 brew install python3
 ```
+* <b>Note:</b>
+Since we currently only support up to Python 3.6.X, you may need to brew install this older Python version. If you have had Python 3.6 installed previously, run the following:
+```shell
+brew switch python 3.6.5_1
+```
+
+* If you need a fresh install of Python 3.6.5, run the following:
+```shell
+brew unlink python
+brew install pkg-config gdbm openssl readline sqlite xz
+brew install --ignore-dependencies https://raw.githubusercontent.com/Homebrew/homebrew-core/f2a764ef944b1080be64bd88dca9a1d80130c558/Formula/python.rb
+```
```

Comment:
Please note somewhere that these instructions apply to Mac OS specifically.

Reply:
Good feedback. I made a commit with updates you suggested. Also may be good to remember that we don't have a documented solution for linux users ATM